task/dev1/randomValuesSelectors/RandomValuesSelector.java

```
@@ -0,0 +1,32 @@
+package task.dev1.randomValuesSelectors;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Select random values
+ */
+public class RandomValuesSelector {
+
+  /**
+   * Select random value from input list
+   * @param valueList - elements for sampling
+   * @param numberOfValues - how many random values we want to get
+   * @return List - list with selected random values
+   */
+  public List select(List valueList, int numberOfValues) {
+    if (valueList == null || valueList.size() < numberOfValues || numberOfValues < 0) {
+      // TODO: add to log a reason
```

good, it's actually better to have todo than silently ignore an error